src/Controller/User/Admin/OverviewController.php

```diff
@@ -17,7 +18,9 @@ class OverviewController extends AbstractController
     #[Breadcrumb('users')]
     public function __invoke(
         Request $request,
-        UserRepository $userRepository
+        UserRepository $userRepository,
+        #[MapQueryParameter]
+        ?int $page = 1
```

Comment:
Does this need to be nullable?

Reply:
fixed 7f77a97a4b13a8d55f43dd97ecb97faffd4eda43